content/rtd.md

```diff
@@ -28,126 +23,115 @@
 
-  **Verify the result**
+**Verify the result**
 
-  That's it! Your site should now be live on
-  http://user-word-count.readthedocs.io (replace project name).
+That's it! Your site should now be live on
+http://youruser-word-count.readthedocs.io (replace project name).
+Note that if your Git repo name contained underscores, they get converted to hyphens.
 
-  **Verify refreshing the documentation**
+**Verify refreshing the documentation**
 
-  Finally, make some changes to your documentation
-  - Add documentation related to other functions
-  - Prerequisites and how to use the program
-  - Rules for contribution
-  - Some example results (figures, tables, ...)
-  - Commit and push them, and verify that the documentation website refreshes after your changes (can take few seconds or a minute)
+Finally, make some changes to your documentation
+- Add documentation related to other functions
+- Prerequisites and how to use the program
+- Rules for contribution
+- Some example results (figures, tables, ...)
+- Commit and push them, and verify that the documentation website refreshes after your changes (can take few seconds or a minute)
 ``````
 
 ```{callout} Do not add the generated build directory to your repository
-
-  The `_build` directory is generated locally with the command `sphinx-build doc _build` and allows you to check the content locally but it should not be part of the Git repository.
-  We recommend to add `_build` to `.gitignore` to prevent you from accidentally adding files below `_build` to the Git repository.
+The `_build` directory is generated locally with the command `sphinx-build
+doc _build` and allows you to check the content locally but it should not be
+part of the Git repository.  We recommend to add `_build` to `.gitignore` to
+prevent you from accidentally adding files below `_build` to the Git
+repository.
 ```
 
-```{callout} Running your own sphinx server
 
-  We recommend to use Read the Docs to host your documentation but if you prefer, you can host your own Sphinx server.
-  If you want to know more about it, look at:
-  - [https://docs.readthedocs.io/en/latest/install.html](https://docs.readthedocs.io/en/latest/install.html)
-  - [https://pypi.org/project/sphinx-autobuild/](https://pypi.org/project/sphinx-autobuild/)
-  - [https://pypi.org/project/sphinx-server/](https://pypi.org/project/sphinx-server/)
+## Alternatives to Read the Docs
 
-```
+Instead of using Read the Docs to host your documentation, you can host your own Sphinx server.
+If you want to know more about it, look at:
+- [https://docs.readthedocs.io/en/latest/install.html](https://docs.readthedocs.io/en/latest/install.html)
+- [https://pypi.org/project/sphinx-autobuild/](https://pypi.org/project/sphinx-autobuild/)
+- [https://pypi.org/project/sphinx-server/](https://pypi.org/project/sphinx-server/)
 
-```{callout} Migrating your own documentation to Sphinx/ Read the Docs
+You can also build Sphinx using GitHub Actions or GitLab CI.
```

Comment:
Should we link to our one example from CodeRefinery?

Reply:
I will add a link.